src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -34,6 +34,9 @@
       "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
   public static final String SQL_LOCAL_USER = "root";
   public static final String SQL_LOCAL_PASSWORD = "";
+
+  // Variables for user login status.
+  public static final int USER_LOGGED_OUT = -1;
```

Comment:
nit: 'USER_LOGGED_OUT' sounds like a boolean. Prefer 'USER_LOGGED_OUT_ID' or 'USER_LOGGED_OUT_DEFAULT_ID' or something similar.

Reply:
Done@

Reply:
Is this constant I created useful, or should I just delete it? It is used on line 82 of this same file, and would be used whenever this functions is called, like in line 57 of `AuthenticationServlet.java`, but instead I could just use '-1'.

Reply:
I think anytime you have an int or string you have to use in multiple places, it's less error-prone and more readable to use a Constant. So yeah I'd say so :)